osquery/CMakeLists.txt

```diff
@@ -98,6 +98,9 @@ ADD_OSQUERY_LINK_CORE("glog")
 ADD_OSQUERY_LINK_CORE("boost_system")
 ADD_OSQUERY_LINK_CORE("boost_filesystem")
 ADD_OSQUERY_LINK_CORE("boost_regex")
+ADD_OSQUERY_LINK_CORE("cryptsetup")
```

Comment:
I don't think the core "libosquery.a" needs to link cryptsetup. Why is this included? The same applies to bz2 and lz4, but I could see either RocksDB or another core library wanting these. However these have not needed core linkage thus far.